src/components/CalendarModal/CalendarModal.module.scss

```
@@ -6,22 +6,66 @@
   height: 100%;
   background: rgba(0, 0, 0, 0.6);
   text-align: center;
-  z-index: 1;
+  z-index: 10;
 }
 
 .modal-main {
   padding: 1rem;
   position: fixed;
   background: white;
-  width: 80%;
+  max-width: 500px;
+  width: 60%;
   height: auto;
   top: 50%;
   left: 50%;
   transform: translate(-50%, -50%);
+  border-radius: 10px;
+}
+
+.modal-main button {
+  background-color: white;
 }
 
 .options {
   display: flex;
+  flex-direction: column;
   gap: 1rem;
   justify-content: center;
+  position: center;
+
+  .apple {
+    background-color: #ffd8d8;
+  }
+
+  .google {
+    background-color: #d4eafb;
+  }
 }
+
+.options button {
+  padding: 1rem;
+  border-radius: 10px;
+  width: 80%;
+  top: 10%;
+  left: 10%;
+  position: relative;
+  display: flex;
+  flex-direction: row;
+  align-items: baseline;
```

```suggestion
  align-items: center;
```

Let's switch to center to align items on the vertical axis